beacon-node-oapi.yaml

```diff
@@ -199,6 +201,9 @@ components:
       $ref: './types/http.yaml#/ErrorMessage'
     IndexedErrorMessage:
       $ref: './types/http.yaml#/IndexedErrorMessage'
+    v2:
```

Comment:
maybe better to group schemas under altair?

Reply:
I figured that v2 might be more than just altair, if the version is moving to a lower level... I don't mind moving it, just not sure?

Reply:
It makes sense for GetBlockResponse but IMO for SignedBeaconBlock (and other objects changed) it would be better to group by fork.

In case of another fork, we would add to v2.GetBlockResponse new fork name and newFork.SignedBeaconBlock

Does that make sense?

Reply:
Yep this sounds good.  Version the API with simple numbers and version the structures with the fork name (internally we could rename the v1 data structures with 'genesis' to make this clearer, but that wouldn't be part of this PR).

Reply:
I think so.

So v2 response objects under a v2 group, and 'altair' objects under an `altair` group? I'll push a change...